app/spdb/build.sbt

```diff
@@ -224,6 +226,37 @@ def swalker(version: Version) = AppConfig(
   )
 ) extending List(with_gogo(version), swalker_credentials(version))
 
+// DNAVARRO
+def dnavarro(version: Version) = AppConfig(
```

Comment:
It compiles and passes the tests for me in `develop` and `release/ocs-2016B` as of e206cff4e16955acac55c2d0b590ee7c222174ae and 4e32d04124288d6cdba067bfaa824cc2bfabe6ac respectively.